### PR TITLE
API Refactor for removal of CMSForm class

### PR DESCRIPTION
**Pull request by tractorcow**
Needed to fix tests at https://github.com/silverstripe/silverstripe-cms/pull/1410

### CODE REVIEW

**Comment by chillu** [comment]
I think this needs the same fix as CMS - we need to declare the phpunit dev dependency explicitly otherwise it takes 3.7, which in turn isn't compatible with the Behat module that's compatible with 4.x...

https://github.com/silverstripe/silverstripe-cms/commit/d9e292ba281342cc7943ae25538daa1d173fa0c6

**Comment by tractorcow** [comment]
Yeah I'm doing it lol.

**Comment by chillu** [comment]
I've sent a similar fix for PHPUnit to https://github.com/silverstripe-labs/silverstripe-reports/pull/27

**Comment by tractorcow** [comment]
GREEN

**Comment by chillu** [comment]
MERGED

File: composer.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
 	"require": {
 		"silverstripe/framework": "^4.0"
 	},
-	"extra": {
-		"installer-name": "siteconfig"
```
Comment (dhensby) [inline composer.json]: What was the reasoning behind removing this?

Comment (tractorcow) [inline composer.json]: `extra` was duplicated in composer.json and was being ignored. Look down the file a bit...

Also it's not needed. `silverstripe/siteconfig` is already named as `siteconfig` and doesn't need an alias.

**Pull request merged**